src/bindings/python/tests/test_graph/test_create_op.py

```diff
@@ -2199,3 +2199,13 @@ def test_interpolate_opset10(dtype, expected_shape, shape_calculation_mode):
     assert node.get_type_name() == "Interpolate"
     assert node.get_output_size() == 1
     assert list(node.get_output_shape(0)) == expected_shape
+
+
+def test_in_nan_opset10():
+    input_shape = [1, 2, 3, 4]
+    input_node = ov.parameter(input_shape, np.float, name="InputData")
+    node = ov_opset10.is_nan(input_node)
+
+    assert node.get_type_name() == "IsNaN"
+    assert node.get_output_size() == 1
+    assert list(node.get_output_shape(0)) == input_shape
```

Comment:
It's worth to check output element type.
```suggestion
    assert node.get_type_name() == "IsNaN"
    assert node.get_output_size() == 1
    assert list(node.get_output_shape(0)) == input_shape
    assert node.get_output_element_type(0) == Type.boolean
```

Reply:
done